v1/talk.yaml

```diff
@@ -0,0 +1,140 @@
+openapi: 3.0.1
+info:
+  version: 0.1.0
+  title: MediaWiki Structured Talk page API
+  description: API for retrieving structured representation of the talk page
+  termsOfService: https://www.mediawiki.org/wiki/REST_API#Terms_and_conditions
+  contact:
+    name: Reading Infrastructure
+    url: https://www.mediawiki.org/wiki/Wikimedia_Reading_Infrastructure_team
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
```

Comment:
These probably need access_check_filter for redirect support? Or it doesn't?

Reply:
Yes.

Reply:
Done.